front/app/containers/Admin/reporting/components/ReportBuilder/TopBar/index.tsx

```diff
@@ -87,6 +111,38 @@ const ContentBuilderTopBar = ({
 
   return (
     <Container>
+      <Modal opened={showQuitModal} close={closeModal}>
```

Comment:
Maybe all this modal stuff could be in a separate component?

Reply:
yeah did that originally, but then i copy how [form builder](https://github.com/CitizenLabDotCo/citizenlab/blob/master/front/app/containers/Admin/formBuilder/components/FormActions/index.tsx) is doing it. And it makes a little sense since its sharing some methods. Can separate it if this is not convincing enough